AIAgent/common/config.py

```diff
@@ -22,7 +23,7 @@ def transform(cls, input: str) -> Path:
 class Platform:
     name: PlatformName
     dataset_configs: list[DatasetConfig] = Field(alias="DatasetConfigs")
-    svms_info: list[SVMInfo] = Field(alias="SVMSInfo")
+    svms_info: list[SVMInfoViaServer] = Field(alias="SVMSInfo")
```

Comment:
It might not be the most descriptive name. I'm open to suggestions.

Reply:
May be more descriptive would be something like "ViaWebSockets" because in this type we use WebSockets to communicate with SVMs. And the other one something like "ViaFileSystem". What do you think?

Reply:
1. "via WebSocket" specifies the communication protocol, whereas "via Server" refers to the architecture employing a client-server model. 
2. I think the "via process" is clearer for the new training process.

Reply:
1. I suppose that creating a "new training process" consists in change of communication protocol.
2. Why does the new process not have the architecture with a client-server model?